snooty.toml

```diff
@@ -1,6 +1,16 @@
 name = "spark-connector"
+title = "MongoDB Spark Connector"
+
+intersphinx = ["https://docs.mongodb.com/manual/objects.inv"]
```

Comment:
Yes! The intersphinx thing lets you use `:binary:`, `:setting:`, `:ref:`, etc. links to references in other projects. I figured I'd put it there just in case you ever needed it (you can add the inventories for the other properties as needed).